e2eTests/e2eTestPrompts.md

```diff
@@ -20,9 +20,9 @@ This file contains prompts used for end-to-end testing to ensure each tool is in
 | azmcp-search-index-list | List all indexes in the Cognitive Search service \<service-name> |
 | azmcp-search-index-list | Show me the indexes in the Cognitive Search service \<service-name> |
 | azmcp-search-index-query | Search for instances of \<search_term> in the index \<index-name> in Cognitive Search service \<service-name> |
-| azmcp-search-list | List all Cognitive Search services in my subscription |
-| azmcp-search-list | Show me the Cognitive Search services in my subscription |
-| azmcp-search-list | Show me my Cognitive Search services |
+| azmcp-search-service-list | List all Cognitive Search services in my subscription |
```

Comment:
Since it's a bit hard to find the exact logic in the big scripts, is there a check to make sure that the E2E test prompts are valid tool names? If not, we should file an issue to add validation for that in CI.

Reply:
We don't currently have logic in place to validate this. I've filed this issue to track adding it: #858.